server/pages/index.jade

```diff
@@ -132,12 +132,16 @@ html(lang=lang, dir=isRTL ? 'rtl' : 'ltr', class=isFluidWidth ? 'is-fluid-width'
 			script(src=urls[ 'manifest' ])
 			script(src=urls[ 'vendor' ])
 			script(src=urls[ jsFile ])
+			if localeFile
+				script(src=urls[ localeFile ])
 			if chunk
 				script(src=urls[ chunk ])
 		else
 			script(src=urls[ 'manifest-min' ])
 			script(src=urls[ 'vendor-min' ])
 			script(src=urls[ jsFile + '-min' ])
+			if localeFile
+				script(src=urls[ localeFile + '-min' ])
```

Comment:
I think it might be a good idea to use a more explicit name for this variable - to convey the fact that this is only data for `moment` and not some more global locale data.